okhttp/src/test/java/okhttp3/internal/http2/HttpOverHttp2Test.java

```diff
@@ -97,6 +98,8 @@
 
 /** Test how HTTP/2 interacts with HTTP features. */
 @RunWith(Parameterized.class)
+@Flaky(issues = {"https://github.com/square/okhttp/issues/4632",
```

Comment:
FWIW, having two issues to track the same flaky test feels bad

Reply:
I think I’d rather track these as normal comments because when the test ceases to be flaky we still probably want the comment link?

Reply:
I don't think it's always two test tracking the bug.  It's often two valid tests failing due to the same bug.  I can move out the issues to comments.

Reification is a big theme for my thinking at the moment, so having it introspectible via code was interesting.  

![image](https://user-images.githubusercontent.com/231923/61268493-841c7780-a793-11e9-91b6-a9905cad8970.png)